### ISSUE TITLE
Error Cannot kill process XXXX: Operation not permitted!

### ISSUE THREAD

**Issue by mdmvgithub** [mind04/mod-ruid2#17]
When Apache 2.4 tries to kill a instance that is running under a userid set by ruid2, it cannot kill process, giving the error Error Cannot kill process XXXX: Operation not permitted!
This happens for example on a PHP script handling Server Event Notifications that is in a loop.
Then other apache clients become affected giving emty responses.

**Comment by arekm** [comment]
Where is the message coming from? I don't see "kill process" in mod_ruid2 sources.

Edit: from php:

./ext/opcache/ZendAccelerator.c:			zend_accel_error(ACCEL_LOG_ERROR, "Cannot kill process %d: %s!", mem_usage_check->l_pid, strerror(errno));

**Comment by arekm** [comment]
Most likely https://bugs.php.net/bug.php?id=76482

**Comment by mdmvgithub** [comment]
ruid2 does not kill process.

Is the master Apache process who periodically tries to kill child 
processes after serving a determinate number of request (set in apache 
conf).

When it tries to kill a process that is running a non-ending PHP script 
(like a notification service), as the process belongs to a different 
user (as ruid2 does), it cannot kill it.

**Comment by arekm** [comment]
Not exactly. It's not apache. It's php (well, it's loaded as so into apache process). The killing code comes from php. php opcache module (which uses kill for some memory management reason). And the problem is that this php code doesn't handle EPERM well.

So this issue should be closed as this is not ruid2 problem.
